docs/installer/lokoctl.md

```diff
@@ -0,0 +1,48 @@
+# lokoctl
+
+lokoctl is a command line interface for Lokomotive, Kinvolk's open-source Kubernetes distribution
+which includes installers for various platforms and Lokomotive components.
+
+## Installation
+
+### Download official releases
+
+To install, find the [appropriate
+package](https://github.com/kinvolk/lokomotive/releases) for your system and
+download it. Lokoctl is packaged as a `tar.gz` archive.
+
+```console
+wget <URL-TO-DOWNLOAD-LOKOCTL>
+```
+After downloading, untar the package. Lokoctl runs as a single binary named `lokoctl`
+
+```console
+tar xvzf <LOKOCTL_TAR_GZ_ARCHIVE>
+```
+
+The final step is to make sure that the `lokoctl` binary is available on the `PATH`
+
+```console
+export PATH=$PATH:/path/to/lokoctl/binary
+```
+
+### Using 'go get'
+
+You can quickly get latest version of `lokoctl` by running following command:
+```
+go get github.com/kinvolk/lokomotive/cmd/lokoctl
```

Comment:
I know this is very trivial but saying that you should have golang installed on the system is important I think.

Reply:
I think this is implied.